changelog/2022-02-28.md

```diff
@@ -0,0 +1,48 @@
+## :mage: February 28, 2022
+
+**CLI version: 0.8.0**
+
+Spring cleaning came early. This release features a lot of backend magic that improves overall stability and UX with Grid. 
+
+We’re also excited to announce a dazzling new enhancement to Sessions. You can now change the instance type of an active Session! That means there’s no need for you to destroy and recreate your Session on a new instance! More details and information on how to use the feature are below.
+
+Upgrade your CLI to `v0.8.0` with `pip install lightning-grid --upgrade`  
+
+Find us in our [Slack Community](http://gridai-community.slack.com) to say hi and/or to express your thoughts/questions.
+
+### Change Instance Type in Sessions
+
+This allows you to upgrade or downgrade the compute capability of the session nodes while keeping all of your work in progress untouched. The following operations are supported:
+
+- Change to a spot instance from an on-demand instance and vice versa
+- Change to a CPU instance from a GPU instance and vice versa
+    
+    <aside>
+    💡 Switching between CPUs and GPUs might require some additional package installation and may increase Session resume time
```

Comment:
Does this happen automatically or does the user need to do anything? Will existing user installed packages need to be reinstalled?

Reply:
@essiequoi has this been tested and validated? Any documentation you can share here?

Reply:
good q stan. @nmiculinic can you weigh in on this?